### DISCUSSION TITLE
From add-products to the MongoDB.

### DISCUSSION THREAD

**Discussion by mjvbz** [mjvbz/eCommerce#4] [General]
## From add-products to the MongoDB.

I think we need should not add our custom images instead of it we can connect them to MongoDB and with add product page we can add our products to the database, and from the database, we can display it in the product images section.

**I think it's better than the last idea...**

**Comment by mjvbz** [comment]
What you think does it will be a good idea?

**Reply by TomsMater** [reply]
yes it would be

**Comment by mjvbz** [comment]
It gets fixed...
